reader/components/toolbar/Toolbar.jsx

```diff
@@ -252,6 +252,7 @@ const Toolbar = ({ viewer, eventBus }) => {
                   },
                 })
               }}
+              onFocus={e => e.target.select()}
```

Comment:
Dirty text select to improve user experience. It allows to remove avoid selecting/manual erasing text to just type another number.